client/app/routes/logout.tsx

```diff
@@ -0,0 +1,10 @@
+import { redirect, type ActionFunctionArgs } from "@remix-run/node";
+import { logout } from "~/lib/logout.server";
+
+export async function action({ request }: ActionFunctionArgs) {
+	return await logout(request);
+}
+
+export async function loader() {
+	return redirect("/discussions");
+}
```

Comment:
Nice, really nice way of doing this. You get it!

Reply:
Thanks, I really appreciate the feedback!